domain-cc/cc-app/src/python_src/api.py

```diff
@@ -88,3 +91,75 @@ def get_classification(claim: Claim) -> Optional[PredictedClassification]:
 
     logging.info(f"classification: {classification}")
     return classification
+
+
+def generate_flattened_claim(contention_text, diagnostic_code, multi_contention_claim):
+    print(f"got dc: {diagnostic_code}")
+    return FlattenedSingleIssueClaim(
+        claim_id=multi_contention_claim.claim_id,
+        form526_submission_id=multi_contention_claim.form526_submission_id,
+        diagnostic_code=diagnostic_code,
+        claim_type=multi_contention_claim.claim_type,
+        contention_text=contention_text,
+    )
+
+
+@app.post("/classifier", deprecated=True)
+def classifier_v1(
+    claim: FlattenedSingleIssueClaim,
+) -> Optional[PredictedClassification]:
+    logging.info(
+        f"claim_id: {claim.claim_id}, form526_submission_id: {claim.form526_submission_id}"
+    )
+    return do_get_classification(claim)
+
+
+@app.post("/classifier/v2")
+def classifier_v2(
+    multi_contention_claim: VaGovClaim,
+) -> ClassifierResponse:
+    contention_log_info = []
+    classifications = []
+    for contention in multi_contention_claim.contentions:
+        contention_text = contention.contention_text
+        diagnostic_code = contention.diagnostic_code
+
+        flattened_single_issue_claim = generate_flattened_claim(
+            contention_text, diagnostic_code, multi_contention_claim
+        )
+        classification = do_get_classification(flattened_single_issue_claim)
+
+        is_in_dropdown = contention_text.strip().lower() in dropdown_values
+        contention_text_no_pii = (
+            contention_text if is_in_dropdown else "Not in dropdown"
+        )
+        contention_log_info.append(
```

Comment:
Should this logging only be for new claims rather than claims for increase? Should the same log statements that are generated in the do_get_classification function be removed to avoid duplicate logs?

Reply:
Yes, we'd ideally remove that, but won't be switching over to new API immediately, so I'll try to figure out a way to preserve old logging formatting before switching over